eng/common/TestResources/New-TestResources.ps1

```diff
@@ -129,6 +128,40 @@ if (!$templateFiles) {
     exit
 }
 
+# If there is a value for $SubscriptionConfiguration look it up and set
+# script-level variables for subsequent steps.
+if ($SubscriptionConfiguration) {
```

Comment:
I was expecting this parameter to be the entire json string not the key to the keyvault to retrieve that string.